man/systemd-suspend.service.xml

```diff
@@ -56,7 +56,7 @@
     <para><filename>systemd-suspend.service</filename></para>
     <para><filename>systemd-hibernate.service</filename></para>
     <para><filename>systemd-hybrid-sleep.service</filename></para>
-    <para><filename>/usr/lib/systemd/system-sleep</filename></para>
+    <para><filename>/usr/lib/systemd/systemd-sleep</filename></para>
```

Comment:
Yes.  The man page is for the binary, not the directory.  To match the NAME section (`<refnamediv>`).

Guess I should have spelled it out in the commit message, instead of just moaning about the confusion :).